src/main/java/snowy/common/ExitCommand.java

```diff
@@ -0,0 +1,21 @@
+package snowy.common;
+
+import snowy.data.SnowyException;
+
+public class ExitCommand extends Command {
+    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "See you next time. Goodbye!";
```

Comment:
Might be a nit on my part, but perhaps naming this constant EXIT_MESSAGE would make the name shorter and easier to reference in the future?